src/rbyte/viz/loggers/rerun_logger.py

```diff
@@ -177,10 +177,17 @@ def _build_columns(
 
                 case rr.Points3D.columns:
                     match (tensor := kwargs[key := "positions"]).shape:
-                        case (3,):
+                        case (2 | 3,):
+                            if tensor.shape == (2,):
+                                kwargs[key] = torch.cat([tensor, torch.zeros(1)])
                             return config.instantiate(**kwargs.cpu().numpy())  # pyright: ignore[reportUnknownMemberType]
 
-                        case (*batch_dims, n, 3):
+                        case (*batch_dims, n, 2 | 3):
```

Comment:
So the point of this dirty hack is that simple `rr.Points2D` displays with y axis down and [currently there is no way to change it](https://github.com/rerun-io/rerun/issues/1387). Displaying in 3D with z=0 looks much better. 

alternating `y=-y` is confusing since it displays wrong values that've already caused me some headache in the past